src/tools/cephfs/Dumper.cc

```diff
@@ -227,23 +227,54 @@ int Dumper::undump(const char *dump_file, bool force)
   char buf[HEADER_LEN];
   r = safe_read(fd, buf, sizeof(buf));
   if (r < 0) {
+    cout << "Error reading " << dump_file << std::endl;
+    VOID_TEMP_FAILURE_RETRY(::close(fd));
+    return r;
+  } else if (r == 0) {
+    //Empty file read
+    cout << "No-op since empty journal dump file " << dump_file << std::endl;
```

Comment:
ditto.

Reply:
Sorry, `derr` here outputs the error text in smithi logs and the test fails. 
I prefer not to grep the smithi logs, to keep it simple.

Reply:
Wouldn't using cout break the uniformity?

Reply:
Yes, but it's fine.